template-files/ios/ExpoKit-Podfile

```diff
@@ -30,12 +30,17 @@ target 'ExpoKitApp' do
     ],
   )
 
+  pod 'Amplitude', :modular_headers => true
+  pod 'CocoaLumberjack', :modular_headers => true
+  pod 'Google-Maps-iOS-Utils', :modular_headers => true
+  pod 'JKBigInteger', :modular_headers => true, :podspec => '../../../ios/vendored/common/JKBigInteger.podspec.json'
+  pod 'MBProgressHUD', :modular_headers => true
```

Comment:
I added `:modular_headers => true` to the pods that CocoaPods was complaining about. I guess we could remove Amplitude and Google Maps from here as they are no longer imported by ExpoKit, but I don't want to change too much and take the risk 😜